core/src/main/java/com/novoda/noplayer/external/exoplayer/text/webvtt/WebvttDecoder.java

```diff
@@ -67,8 +68,13 @@ protected WebvttSubtitle decode(byte[] bytes, int length, boolean reset)
     definedStyles.clear();
 
     // Validate the first line of the header, and skip the remainder.
-    WebvttParserUtil.validateWebvttHeaderLine(parsableWebvttData);
-    while (!TextUtils.isEmpty(parsableWebvttData.readLine())) {}
+    try {
+      WebvttParserUtil.validateWebvttHeaderLine(parsableWebvttData);
+    } catch (ParserException e) {
+      throw new SubtitleDecoderException(e);
+    }
+    while (!TextUtils.isEmpty(parsableWebvttData.readLine())) {
+    }
```

Comment:
Should we be doing something here? 🤔 I was gonna suggest a `while(...);` but maybe a comment instead?

Reply:
This is a copy from the `exo-player` version, we try not to change too much to avoid issues later 👅